ee/agent/storage/bbolt/stats_bbolt.go

```diff
@@ -0,0 +1,112 @@
+package agentbbolt
+
+import (
+	"encoding/json"
+	"fmt"
+	"log/slog"
+
+	"go.etcd.io/bbolt"
+)
+
+type bucketStatsHolder struct {
+	Stats        bbolt.BucketStats
+	FillPercent  float64
+	NumberOfKeys int
+	Size         int
+}
+
+type dbStatsHolder struct {
+	Stats bbolt.TxStats
+	Size  int64
+}
+
+type Stats struct {
```

Comment:
```suggestion
type stats struct {
```

I think? Or maybe even make it anonymous in the `GetStats` method?

Reply:
yeah i'll get that cleaned up, ty!